Update Preview dialog to be less confusing

PatoBeltran opened the pull request:
fixes https://github.com/NuGet/Home/issues/5641

Commit:
Update Preview dialog to be less confusing

dnfclas commented:
@PatoBeltran,
Thanks for having already signed the Contribution License Agreement. Your agreement was validated by .NET Foundation. We will now review your pull request.
_Thanks,
.NET Foundation Pull Request Bot_

nkolev92 commented:
@PatoBeltran Can you paste images of the dialog before and after the change?

karann-msft commented on src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PreviewWindow.xaml:
```diff
@@ -31,14 +31,8 @@
-      FontSize="{Binding ElementName=_self,Path=FontSize,Converter={StaticResource Font122PercentSizeConverter}}" />
-
-    <TextBlock
       Grid.Row="1"
```
Since, we no longer have the first text block, we should consider moving this text block to Grid.Row = "0" and move any other items below up by one as well.

PatoBeltran commented:
@nkolev92 here is how the dialog looks with and without this change.

New behavior
![newbehavior](https://user-images.githubusercontent.com/2132567/29133023-f665e06c-7ce6-11e7-847f-02a976a584e8.PNG)

Old behavior
![oldbehavior](https://user-images.githubusercontent.com/2132567/29133026-f8f9a750-7ce6-11e7-8ee8-8cab14dcdce8.PNG)

@karann-msft addressed comment

emgarten reviewed (approved):
LGTM

karann-msft commented:
:shipit:
